docker-compose.yml

```diff
@@ -142,7 +142,7 @@ services:
       OPENMRS_HOSTNAME: "openmrs-tomcat"
       OPENMRS_USERNAME: "admin"
       OPENMRS_PASSWORD: "${OPENMRS_PASSWORD:-Admin123}"
-      BAHMNI_LOGIN_URL: "http://localhost/"
+      BAHMNI_LOGIN_URL: "http://localhost/bahmni/home/index.html#/login"
```

Comment:
this line is not necessary. this creates conflicts with the upstream. Could you fix it?

Reply:
ah yes :)